
In order to build on older platform images that are missing
  9812 headers should be free of SUS compiler tyranny
when building in a 18.4 64-bit dev zone, we do not define
POSIX_SOURCE.

--- a/libiberty/sigsetmask.c	2013-11-04 15:33:40.000000000 +0000
+++ b/libiberty/sigsetmask.c	2019-08-22 13:09:24.815534431 +0000
@@ -15,7 +15,15 @@
 
 */
 
-#define _POSIX_SOURCE
+/* 
+ * In order to build on older platforms that are missing the fix for
+ * 9812 headers should be free of SUS compiler tyranny
+ * we uncomment the define for _POSIX_SOURCE
+ *
+ * #define _POSIX_SOURCE
+ *
+ */
+
 #include <ansidecl.h>
 /* Including <sys/types.h> seems to be needed by ISC. */
 #include <sys/types.h>
